libs/core/src/lib/split-button/split-button.component.ts

```diff
@@ -79,9 +85,63 @@ export class SplitButtonComponent {
     @ContentChild(MenuComponent)
     menu: MenuComponent;
 
+    /** @hidden */
+    private _firstElement: MenuItemComponent;
+
+    get firstElement(): MenuItemComponent {
+        return this._firstElement;
+    }
+
+    set firstElement(item: MenuItemComponent) {
+        this._firstElement = item;
+    }
+
+    /** An RxJS Subject that will kill the data stream upon component’s destruction (for unsubscribing)  */
+    private readonly _onDestroy$: Subject<void> = new Subject<void>();
+
+    constructor(private _changeDetectorRef: ChangeDetectorRef) { }
+
     /** @hidden Emits event when main button is clicked */
     onMainButtonClick(event: MouseEvent): void {
-        this.primaryButtonClicked.emit(event);
+        this.mainButtonClicked();
         event.stopPropagation();
     }
+
+    ngAfterContentChecked(): void {
```

Comment:
First declare life cycle methods then other public methods.
I recommend following our unofficial but popular convention of naming methods responsible for setting up observers/listeners like: `selectedSubscribed` -> `listenOnSelectedAction`